docs/content/installation/env_vars.en.md

```diff
@@ -53,30 +52,28 @@ $ export FISSION_ROUTER=$(kubectl --namespace fission get svc router -o=jsonpath
 $ export FISSION_ROUTER=$(kubectl --namespace fission get svc router -o=jsonpath='{..ip}')
 ```
 
-### Using FISSION_ROUTER env var
-
-```bash
-$ curl http://${FISSION_ROUTER}/<url-path>
-```
-
-### Troubleshooting
-
-If your cluster is running in an environment that does not support external load balancer (e.g., minikube), the EXTERNAL-IP of fission router will stay in pending state.
+If your cluster is running in an environment that does not support external load balancer (e.g., minikube), 
+the EXTERNAL-IP of fission router will stay in pending state.
 
 ```bash
 $ kubectl --namespace fission get svc router
 NAME      TYPE           CLUSTER-IP     EXTERNAL-IP   PORT(S)        AGE
 router    LoadBalancer   10.39.253.73   <pending>   80:31377/TCP   27d
 ```
 
-In this case, you can use the port-forward method instead:
+In this case, you can change the service type to `NodePort` and use the `port-forward` method instead:
```

Comment:
Why? Port-forwarding doesn't need nodeport, right?

Reply:
The original idea is that it may confuse people seeing a service stay in `<pending>` state. That's why I suggest changing service type to NodePort here.